MapboxNavigation/NavigationViewController.swift

```diff
@@ -147,6 +147,11 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
      */
     public var shouldManageApplicationIdleTimer = true
     
+    /**
+     Coordinate which is used to determine what associated building extrusion should be highlighted.
+     */
+    public var buildingExtrusionCoordinate: CLLocationCoordinate2D? = nil
```

Comment:
Since a single route can have multiple destinations, we’ll need to refactor this code so that NavigationViewController exposes only a feature flag and the logic to choose a coordinate lives inside NavigationMapView (based on a waypoint or route progress being passed in).

Reply:
I'm not sure we can get away with feature flag to do this. Since in most cases route destination coordinate is not equal to highlighted building coordinate we'll have to store highlighted building coordinates separately. In such case probably list with separate coordinates will have to be used (along with route which is passed to `NavigationViewController`). Or am I missing something here?